ray-operator/config/samples/ray-service.autoscaler.yaml

```diff
@@ -9,72 +9,56 @@ metadata:
 spec:
   serviceUnhealthySecondThreshold: 300 # Config for the health check threshold for service. Default value is 60.
   deploymentUnhealthySecondThreshold: 300 # Config for the health check threshold for deployments. Default value is 60.
-  serveConfig:
-    importPath: fruit.deployment_graph
-    runtimeEnv: |
-      working_dir: "https://github.com/ray-project/test_dag/archive/41d09119cbdf8450599f993f51318e9e27c59098.zip"
-    deployments:
-      - name: MangoStand
-        numReplicas: 1
-        userConfig: |
-          price: 3
-        rayActorOptions:
-          numCpus: 1
-      - name: OrangeStand
-        numReplicas: 1
-        userConfig: |
-          price: 2
-        rayActorOptions:
-          numCpus: 0.1
-      - name: PearStand
-        numReplicas: 1
-        userConfig: |
-          price: 1
-        rayActorOptions:
-          numCpus: 0.1
-      - name: FruitMarket
-        numReplicas: 1
-        rayActorOptions:
-          numCpus: 0.1
-      - name: DAGDriver
-        numReplicas: 1
-        routePrefix: "/"
-        rayActorOptions:
-          numCpus: 0.1
+  serveConfigV2: |
+    applications:
+      - name: app1
+        import_path: blocked.app
+        route_prefix: /
+        runtime_env:
+          working_dir: "https://github.com/zcin/sample-serve-applications/archive/refs/heads/main.zip"
```

Comment:
Can you move this application to [ray-project](https://github.com/ray-project) instead of your personal GitHub? I'd suggest either making a new repo there or using [this one](https://github.com/ray-project/serve_workloads).

Reply:
And could you use a specific commit hash instead of linking to `main`? That prevents the test from failing if someone pushes a broken change to the repo.

Reply:
Thanks for the suggestion! I've pushed it to the repo you linked.

Reply:
Great, thanks!